koap/build.gradle.kts

```diff
@@ -11,7 +11,14 @@ apply(from = rootProject.file("gradle/jacoco.gradle.kts"))
 
 kotlin {
     jvm()
-    js().browser()
+    js {
+        browser {
+            webpackTask {
+                output.libraryTarget = "commonjs2"
```

Comment:
Why the switch to `commonjs2`?

I was under the impression that UMD (which I think we were configured with by default) is compatible with CommonJS?

Reply:
It was something that @davidtaylor-juul had me do. I can't precisely remember the reason.